src/three-domain-secure/component.test.js

```diff
@@ -113,24 +138,65 @@ describe("three domain secure component - isEligible method", () => {
   });
 
   test("catch errors from the API", async () => {
-    const mockRequest = vi.fn().mockRejectedValue(new Error("Error with API"));
-    const threeDomainSecureClient = createThreeDomainSecureComponent({
-      request: mockRequest,
-    });
+    mockRestClient.request = vi
+      .fn()
+      .mockRejectedValue(new Error("Error with API"));
+    const threeDomainSecureClient = createThreeDomainSecureComponent();
 
     expect.assertions(2);
     await expect(() =>
       threeDomainSecureClient.isEligible(defaultMerchantPayload)
     ).rejects.toThrow(new Error("Error with API"));
-    expect(mockRequest).toHaveBeenCalled();
+    expect(mockRestClient.request).toHaveBeenCalled();
   });
 });
 
-describe("three domain descure component - show method", () => {
-  test.todo("should return a zoid component", () => {
+describe.todo("three domain descure component - show method", () => {
```

Comment:
this is still `todo`. is this intentional?

Reply:
oh wow i didn't know you could do that